packages/dd-trace/src/profiling/exporters/agent.js

```diff
@@ -89,6 +89,22 @@ function computeRetries (uploadTimeout) {
   return [tries, Math.floor(uploadTimeout)]
 }
 
+const libuvThreadPoolSize = (() => {
+  const ss = process.env.UV_THREADPOOL_SIZE
+  if (ss === undefined) {
```

Comment:
Sorry for the noise, I did not understand that you were delegating to the backend the computation of the default value.
My first take was that the library was in a better position to determine the default value, but since the node version is part of profile metadata, it might be better to let the backend handle this, since updating code in the backend is far more easier.